circom-prover/Cargo.toml

```diff
@@ -81,10 +81,10 @@ ark-poly = { version = "=0.4.1", default-features = false, features = [
 rand = { version = "0.8", features = ["std"] }
 
 # witnesscalc-adapter
-witnesscalc-adapter = { version = "0.1.0", optional = true }
+witnesscalc-adapter = { git = "https://github.com/zkmopro/witnesscalc_adapter.git", branch = "dylib", optional = true }
```

Comment:
Curious what the difference betwee `0.1.0` and `dylib` branch?

Reply:
`uniffi` needs to link to dylib https://github.com/zkmopro/witnesscalc_adapter/pull/12/files
I will publish a new version of `witnesscalc-adapter` if the e2e tests pass